frontend/src/validators/expense.js

```diff
@@ -28,7 +29,7 @@ const schema = Joi.object().keys({
     .label('IncurredAt'),
   category: Joi.string().required()
     .label('Category'),
-  payoutMethod: Joi.string().valid(pluck(payoutMethods('en'), 'value'))
+  payoutMethod: Joi.string().valid(pluck(payoutMethods(i18n('en')), 'value')) // i18n.getString not used here
```

Comment:
`payoutMethods()` originally took `lang` and imported i18n on its own. That's worse because we don't always have `lang` available (like in cases where a subcomponent doesn't know about Collective, so can't access lang). I changed it to pass in i18n.

This is an edge case - string doesn't matter, joi only checks for `value`. Refactoring everything to avoid this one hacky line didn't seem worth it.